lib/pbench/server/auth/__init__.py

```diff
@@ -366,15 +388,18 @@ def _get(self, path: str, **kwargs) -> requests.Response:
                 verify=self.verify,
             )
         except Exception as exc:
-            self.logger.exception("{}", str(exc))
-            raise OidcError(
+            self.logger.exception(
+                "Exception submitting a GET request for OIDC client {}", self.__repr__()
```

Comment:
The wording is a bit ambiguous; most likely the exception is raised by the decode of the returned HTTP status rather than by a failure "submitting" the request (and the word "submitting" in this context is a bit vague anyway). Maybe just "GET request failed"??

Which actually brings up another question ... is there something in your `self.connection` setup that's causing `get` to `raise_from_status()`? Are we actually getting exceptions from HTTP return statuses here?

Reply:
Added `raise_for_status` as a hook for `requests.session()` object.